demo/src/screens/componentScreens/TextFieldScreen/BasicTextFieldScreen.js

```diff
@@ -57,6 +64,8 @@ export default class BasicTextFieldScreen extends Component {
       error
     } = this.state;
 
+    
+
```

Comment:
Not needed

Reply:
I think it's a github issue.. 
anyway I remove those lines, maybe it'll fix it..